src/loaders/ObjectLoader.js

```diff
@@ -451,9 +451,10 @@ Object.assign( ObjectLoader.prototype, {
 
 	},
 
-	parseMaterials: function ( json, textures ) {
+    	parseMaterials: function ( json, textures ) {
```

Comment:
Can you please correct the formatting in this line?

Reply:
done